torch/_functorch/aot_autograd.py

```diff
@@ -1568,10 +1797,15 @@ def aot_dispatch_base_graph(flat_fn, flat_args: List[Tensor], aot_config: AOTCon
     if aot_config.enable_log:
         aot_graphs_log.info("%s", lazy_format_graph_code("Forward graph", fw_module, aot_config.aot_id))
 
-    return fw_module
+    # TODO: should factor this into a separate function for export that always only returns just the graph.
+    if aot_config.is_export:
+        assert maybe_subclass_meta is None, "aot_export_module does not support tensor subclass inputs for now."
```

Comment:
this seems fine

Reply:
if you know how to do the update, I'd write that plan down so that someone else can do it later